webpack.config.js

```diff
@@ -160,7 +161,10 @@ if ( calypsoEnv === 'desktop' ) {
 
 const jsLoader = {
 	test: /\.jsx?$/,
-	exclude: /node_modules/,
+	include: [
+		path.resolve( './client' ),
+		fs.realpathSync( './node_modules/notifications-panel' )
```

Comment:
We don't need to provide a minified bundle. We can release a `es5` transpiled version. That should fix babel problems with symbolic links.